doc/release-notes.md

```diff
@@ -102,9 +102,12 @@ will trigger BIP 125 (replace-by-fee) opt-in. (#11413)
   option `-deprecatedrpc=banscore` is used. The `banscore` field will be fully
   removed in the next major release. (#19469)
 
-- The `testmempoolaccept` RPC returns `vsize` and a `fee` object with the `base` fee
+- The `testmempoolaccept` RPC returns `vsize` and a `fees` object with the `base` fee
   if the transaction passes validation. (#19940)
 
+- The `testmempoolaccept` RPC returns `max-fee-exceeded` rather than `absurdly-high-fee`
```

Comment:
Given that the criteria to throw this reject reason appears to be the exceeding of a _feerate_, would it not be more informative to make the `reject-reason` `max-feerate-exceeded` instead of `max-fee-exceeded`? I realize that this would have been a better feedback for #19339, but I only see it now.

Reply:
Thanks for reviewing :) my "inspiration" was `TransactionError::MAX_FEE_EXCEEDED`, hopefully not too big of a deal

Reply:
It's a bit of a pet-peeve of mine that *fees* and *feerates* are often mixed up in Bitcoin, but the ship has probably sailed for this change? :thinking:
I'll have to be faster next time. ;)

Reply:
@Xekyo You're absolutely right that they're often mixed up, but in this context it is actually referring to an absolute fee (by default 0.1 BTC...) rather than a feerate.

Reply:
Okay, thanks for the clarification. I'm a bit confused by this description then:

>  The `sendrawtransaction` error code for exceeding `maxfeerate` has been changed from  `-26` to `-25`. The error string has been changed from "absurdly-high-fee" to  "Fee exceeds maximum configured by user (e.g. -maxtxfee, maxfeerate)." The  `testmempoolaccept` RPC returns `max-fee-exceeded` rather than `absurdly-high-fee`  as the `reject-reason`. (#19339)

It sounds to me that exceeding `maxfeerate` causes the `reject-reason` `max-fee-exceeded`. If they are referring to two separate error resolutions, it wasn't apparent to me. I'm not overtly invested, though, if that looks fine to more experienced contributors.

Reply:
Oh perhaps it is me who is confusing things.

Reply:
The [`maxfeerate`](https://github.com/bitcoin/bitcoin/blob/12a1c3ad1a43634d2a98717e49e3f02c4acea2fe/src/rpc/rawtransaction.cpp#L889) passed into testmempoolaccept is a feerate and the wallet [`-maxtxfee`](https://github.com/bitcoin/bitcoin/blob/12a1c3ad1a43634d2a98717e49e3f02c4acea2fe/src/wallet/init.cpp#L54) option is a fee amount. I'm not really sure why this is the case 😅